infra/mysql/schema/2022061801-messengers-create-notifications.sql

```diff
@@ -0,0 +1,16 @@
+CREATE SCHEMA IF NOT EXISTS `messengers` DEFAULT CHARACTER SET utf8mb4;
+
+CREATE TABLE `messengers`.`notifications` (
+  `id`           VARCHAR(22)  NOT NULL,          -- お知らせID (Primary Key用)
+  `created_by`   VARCHAR(22)  NOT NULL,          -- 登録者ID
+  `creator_name` VARCHAR(32)  NOT NULL,          -- 登録者名
+  `updated_by`   VARCHAR(22)  NOT NULL,          -- 更新者ID
+  `title`        VARCHAR(128) NOT NULL,          -- タイトル
+  `body`         TEXT         NOT NULL,          -- 本文
+  `since`        DATETIME     NOT NULL,          -- 掲載開始時間
+  `until`        DATETIME     NOT NULL,          -- 掲載終了時間
```

Comment:
@hamachans 

他のアプリ触ってて思ったんだけど、もしかしてお知らせって掲載開始日時だけあれば良さそう...?👀
(一度掲載されたお知らせってずっと見れる気がして)

もしその認識であってそうであれば、 `since`, `until` みたいなのやめて `publishedAt` みたいなの1つ持たせる感じでもいいのかなと思ったので質問🙇‍♂️

Reply:
あー、それでもいいかもしれないね、、
ただあるか分からないけどある一定の期間だけ見せる時とか、掲載終了必要な気がするな
もしないなら表示非表示を切り替える導線必要な気がしているんだけどどうだろう？

Reply:
終了期限つけるか、公開フラグのどちらかって感じか

Reply:
公開フラグをつける実装でお願いします！

Reply:
https://www.figma.com/file/CdLAwdezC2v4thuj4tKBSU/Online-Marche?node-id=907%3A635
デザインこんな感じかな、理解できそう？

Reply:
承知しました！デザイン修正ありがとう！